include/svs/index/vamana/vamana_build.h

```diff
@@ -219,6 +219,13 @@ class VamanaBuilder {
         size_t num_batches = std::max(
             size_t{40}, lib::div_round_up(num_nodes, lib::narrow_cast<size_t>(64 * 64))
         );
+
+        // If num_batches is greater than num_nodes (num_batches will be 40), set
```

Comment:
The reason is clear without the specific number. Plus, if we ever change `40` in the code, we will likely forget to change it in this comment. :)
```suggestion
        // If num_batches is greater than num_nodes, set
```

Reply:
That makes sense. It is indeed different from HNSWlib, so it should not have the same name.